extensions/cornerstone/src/commandsModule.ts

```diff
@@ -127,6 +128,69 @@ function commandsModule({
   }
 
   const actions = {
+    hydrateSecondaryDisplaySet: async ({ displaySet, viewportId }) => {
+      if (!displaySet) {
+        return;
+      }
+
+      // if segmentation, we can't use the isOverlay, since PMAP is also an overlay but not a segmentation
```

Comment:
Is there some way to add something to the display set that indicates this is a segmentation - I really dislike relying on the modality field as that embeds knowledge about the display set parsing details into the commands module.